src/components/idleMembers/card/index.tsx

```diff
@@ -10,13 +10,15 @@ type Props = {
 }
 
 const Card: FC<Props> = ({ idleMemberUserName }) => {
+  const [isError, setIsError] = useState(false);
```

Comment:
NIT: Could we think of a better variable aligning with the Image and source?
I see that we are utilizing this for Image Error scenario, so something on that line :)